src/lib/helpers/shareDomainAsset.ts

```diff
@@ -0,0 +1,30 @@
+export enum NFT_ASSET_SHARE_KEYS {
+	TWITTER = 'TWITTER',
+}
+
+export const NFT_ASSET_SHARE_OPTIONS = {
+	[NFT_ASSET_SHARE_KEYS.TWITTER]: {
+		URL: 'https://twitter.com/share?url=NFT_ASSET_DOMAIN_URL',
+		OPTIONS:
+			'menubar=no,toolbar=no,resizable=no,scrollbars=no,personalbar=no,height=575,width=500',
+	},
+};
+
+export const shareDomainAsset = (
+	zna: string,
+	key: NFT_ASSET_SHARE_KEYS = NFT_ASSET_SHARE_KEYS.TWITTER,
+): void => {
+	const domainUrl = encodeURIComponent(
+		// TODO: update with env var
+		`https://zns-share.herokuapp.com/${zna}`,
```

Comment:
Requires the following:

New env variable in zOS .env / .env.sample for this use.

`REACT_APP_ZNS_SHARE_BASE_URL=`

Add the variable to our Netlify site settings here: https://app.netlify.com/sites/zapp-nft/settings/env

And use in zApp-NFTs like so: `process.env.REACT_APP_ZNS_SHARE_BASE_URL`